benchmark/framework/aea_test_wrapper.py

```diff
@@ -22,11 +22,12 @@
 from threading import Thread
 from typing import Dict, List, Optional, Tuple, Type, Union
 
+from fetchai_crypto import FetchAICrypto
```

Comment:
why don't we use `make_crypto`?...doesn't matter. Just wondering what policy we adopt